### ISSUE TITLE
Unable to unset private variable

### ISSUE THREAD

**Issue by bcdonadio** [tsuru/tsuru#1589]
When you set a private variable, you're unable to unset it until you overwrite it with a public variable and then try to unset it again.

Steps to reproduce:
```bash
tsuru env-set -a app test1=x
tsuru env-get -a app #test1 exists as expected
tsuru env-unset -a app test1
tsuru env-get -a app #test1 doesn't exists as expected

tsuru env-set -a app -p test2=x
tsuru env-get -a app #test2 exists as expected
tsuru env-unset -a app test2
tsuru env-get -a app #test2 STILL EXISTS, unexpectedly

tsuru env-set -a app test2=x
tsuru env-unset -a app test2
tsuru env-get -a app #now test2 is properly removed
```

Using tsurud 1.2.2.

**Comment by andrestc** [comment]
I was able to reproduce this on 1.3.0-rc3. Thanks for reporting.

**Comment by cezarsa** [comment]
Fixed by tsuru/tsuru#1625.
